### ISSUE TITLE
Unable to upload file via keyboard while creating a post

### ISSUE THREAD

**Issue by krallowski** [humhub/humhub#8269]
### What steps will reproduce the problem?
1. Write a new post
2. Press tab to reach the "Upload File" Icon button
and / or
1. Write a new post
2. Press tab to reach the arrow next to the upload file icon button
3. Press enter to open the menu
4. Press tab or arrow keys to reach the menu elements

### What is the expected result?
The icon button or the menu items get focused to upload a file

### What do you get instead?
I cannot reach the icon button neither the menu items

### Additional info
The screenshot show the button and menu items not being part of the tab order.
The missing href attribute for the links as menu items might lead to not being focused by keyboard.

| Q                | A
| ---------------- | ---
| HumHub version   | 1.18

<img width="690" height="254" alt="Image" src="https://github.com/user-attachments/assets/62138ee4-27bd-4825-9c10-712c2ea3c193" />

| PHP version      | 
| Operating system | Linux > Firefox browser

> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.
> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.

**Comment by krallowski** [comment]
At least the issue with accessing the button via keyboard is fixed with 1.19 (https://github.com/humhub/humhub/pull/8364)

**Comment by yurabakhtin** [comment]
@luke- https://github.com/humhub/humhub/pull/8437
